components/avatar/avatar.css

```diff
@@ -0,0 +1,69 @@
+@import "../../css/common.css";
+
+.avatar{
+
+}
+/* selecting img element immediate inside .avatar class*/
+.avatar-round > img {
+    border-radius: 50%;
+    height: 10rem;
+    margin: 1rem;
+}
+
+
+.avatar-square > img {
+    border-radius: 7px;
+    height: 10rem;
+    margin: 1rem;
+}
+
+/* avatar sizes */
+.avatar-md > img{
+    height: 7rem;
+}
+
+.avatar-sm > img{
+    height: 5rem;
+}
+
+.avatar-xs > img{
+    height: 3rem;
+}
+
+/* avatar badges */
+
+.avatar-online{
+    z-index: 1;
+    position: absolute;
+    background-color: var(--green-500);
+    border: 2px solid white;
+    height: 1.5rem;
+    width: 1.5rem;
+    border-radius: 50%;
+    bottom: 1.5rem;
+    left: 8rem;  
+}
+
+.avatar-dnd{
+    z-index: 1;
+    position: absolute;
+    background-color: var(--red-500);
+    border: 2px solid white;
+    height: 1.5rem;
+    width: 1.5rem;
+    border-radius: 50%;
+    bottom: 1.5rem;
+    left: 8rem;  
+}
+
+.avatar-offline{
+    z-index: 1;
+    position: absolute;
+    background-color: var(--black-500);
+    border: 2px solid white;
+    height: 1.5rem;
+    width: 1.5rem;
+    border-radius: 50%;
+    bottom: 1.5rem;
+    left: 8rem;
+}
```

Comment:
.avatar-online,  .avatar-dnd and .avatar-offline for these classes you are repeating the same code. You only have to change the background color so just add a new class for that.